### DISCUSSION TITLE
End of suppor for 3.9

### DISCUSSION THREAD

**Discussion by dreiucker** [rabbitmq/rabbitmq-server#4363] [Other] [answered]
RabbitMQ 3.10 was announced to be shipped on 11th April: https://blog.rabbitmq.com/posts/2022/03/rabbitmq-3.10.0-release-calendar/

With that I'm wondering when the end of service of 3.9 is planned. Before when it was shipped on 2 year basis the support for the service lasted 34 months. However, now 3.10 is shipped not even one year after 3.9. As a result I would assume that the supported time of service for 3.9 will therefore be shorter. Would it be reasonable to assume that the support for 3.9 ends around Q4 this year and the extended support around Q2 next year?

And yes I'm sorry I'm bugging you about that topic before it is officially released. I'm not looking for straight dates but rather a rough estimate to integrate the RMQ EOL into enterprise processes. 

 https://www.rabbitmq.com/versions.html

**Comment by michaelklishin** [comment]
It hasn't been decided. Support cycle will be shorter because we plan to eventually start releasing a new minor every six months. But we haven't discussed how much shorter it's going to be and there may be legal reasons affecting this decision.

**Comment by dcarwin-pivotal** [comment, marked as answer]
Hi Monika-Lydia,

We adhere to a support policy as follows: 
- Each minor release receives a minimum of 12 months of general support and six months of extended support - so life span of a minor is on the order of 18 months. it can be longer but not shorter. 
- When we ship a new minor, we set the clock on the end of general support for the current minor, ensuring six months of support overlap to perform an upgrade.

**Reply by dreiucker** [reply]
I assume that with ensuring six months of support overlap we are talking about the full support and then 6 months extended support after that period.

**Reply by michaelklishin** [reply]
Not quite. It means that when 3.10 comes out, 3.9 will be supported for at least 6 months. In other words, 3.9 support doesn't immediately go away when 3.10 ships.

**Comment by michaelklishin** [comment]
If we support every minor for 18 months but keep churning them out every 6, we will have to support half a dozen of series in parallel at some point in the next couple of years. That's not sustainable, so I expect this policy to change one way or another in the future. 

But for 3.9 and likely 3.10, we will adhere to the status quo @dcarwin-pivotal described. Thanks, Dan!
